yolox/evaluators/coco_evaluator.py

```diff
@@ -27,9 +27,33 @@
 )
 
 
+def per_class_AR_table(coco_eval, class_names=COCO_CLASSES, headers=["class", "AR"], colums=6):
```

Comment:
`per_class_AR_table is similar` to `per_calss_AP_table`, why not merge them into one function?

Reply:
First. At that time, in order to quickly test AP and AR under different conditions, such as different IoU and different regional range indicators.
Second. I don't want to study the third-party library of tabulation too much.
Third. In actual projects, AP and AR are equally important, and the use of two functions reflects the same importance.
Of course, I may change some of the above thoughts later, but now it is very flexible to use. (Later, may be changed to AP, AR, create_table three functions)